### ISSUE TITLE
JABBA Texture

### ISSUE THREAD

**Issue by superzanti** [superzanti/TheVerse#12]
Starting on line 17644 of fml-client-latest.log

```
[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

[15:34:14] [Client thread/WARN] [JABBA/]: JABBA-Debug Problem loading texture: extrautils:bw_(obsidian)

```

**Comment by CRezent** [comment]
What I found online:

This error is actually harmless, It only indicates that the icon for an upgrade material could not be loaded, so Jabba is defaulting to black. It is unrelated to blank/missing textures. This portion is being re-written so that if an icon isn't loaded, Jabba will try the next entry in the OreDictionary until an icon is loaded, then default to black if still unable to load.

I looked around for a while and couldn't find anything off the error code from google.
